subsys/bluetooth/controller/hci/hci.c

```diff
@@ -6425,17 +6432,28 @@ static void le_big_sync_established(struct pdu_data *pdu,
 	sep->irc = lll->irc;
 	sep->max_pdu = sys_cpu_to_le16(lll->max_pdu);
 	sep->iso_interval = sys_cpu_to_le16(lll->iso_interval);
-	sep->num_bis = lll->num_bis;
+	sep->num_bis = lll->stream_count;
```

Comment:
What is the difference between the `stream_count` and the `num_bis` in the `lll_sync_iso`?

Reply:
Yes. For now the controller listens to all BIS subevents, but subsequent PRs implementation will be improved to only listen to selected subevents.